packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -207,23 +206,8 @@ const Loader = React.memo(
   }
 );
 
-const TrackedLoader = withAITracking(
-  userAI.reactPlugin,
-  Loader,
-  "IModelLoader",
-  "tracked-loader"
-);
-
-type IModelLoaderProps = ModelLoaderProps & {
-  appInsightsKey?: string;
-};
-
-const IModelLoader = (props: IModelLoaderProps) => {
-  if (props.appInsightsKey) {
-    return <TrackedLoader {...props} />;
-  } else {
-    return <Loader {...props} />;
-  }
+const IModelLoader = (props: ModelLoaderProps) => {
```

Comment:
we dont need this wrapped component anymore, can just use Loader, but i guess you can rename it to IModelLoader.

@kckst8, preference on defaults vs named export?

Reply:
since this file has a single export with no real potential for others and its "internal", I typically prefer default. That said, its not a strong preference and I wouldn't argue named exports